src/azul/__init__.py

```diff
@@ -65,6 +65,16 @@ def url_redirect_base_domain_name(self) -> str:
     def es_timeout(self) -> int:
         return int(os.environ['AZUL_ES_TIMEOUT'])
 
+    @property
+    def data_browser_endpoint(self):
```

Comment:
The way we use the term, an "endpoint" is a URL prefix. This method returns a domain name. Rename to `data_browser_domain`